docs/Versions.md

```diff
@@ -17,7 +17,7 @@ Versioned formulae we include in [homebrew/core](https://github.com/homebrew/hom
 * Submitted versioned formulae should be expected to be used by a large number of people. If this ceases to be the case, they will be removed. We will aim not to remove those in the [top 3,000 `install_on_request` formulae](https://brew.sh/analytics/install-on-request/).
 * Versioned formulae should not have `resource`s that require security updates. For example, a `node@6` formula should not have an `npm` resource but instead rely on the `npm` provided by the upstream tarball.
 * Versioned formulae should be as similar as possible and sensible compared to the main formulae. Creating or updating a versioned formula should be a chance to ask questions of the main formula and vice versa, e.g. can some unused or useless options be removed or made default?
-* No more than five versions of a formula (including the main one) will be supported at any given time, regardless of usage. When removing formulae that violate this, we will aim to do so based on usage and support status rather than age.
+* No more than five versions of a formula (including the main one) will be supported at any given time, unless they have over 1000 [analytics 30 days installs](https://formulae.brew.sh/analytics/install/30d/) of usage. When removing formulae that violate this, we will aim to do so based on usage and support status rather than age.
```

Comment:
```suggestion
* No more than five versions of a formula (including the main one) will be supported at any given time, unless they have more than 1000 [analytics 30 days installs](https://formulae.brew.sh/analytics/install/30d/) of usage. When removing formulae that violate this, we will aim to do so based on usage and support status rather than age.
```